fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckBottomSheet.kt

```diff
@@ -85,19 +73,17 @@ fun ReviewQualityCheckBottomSheet(
                     onProductRecommendationsEnabledStateChange = {
                         store.dispatch(ReviewQualityCheckAction.ToggleProductRecommendation)
                     },
-                    onReviewGradeLearnMoreClick = { url ->
+                    onReviewGradeLearnMoreClick = {
                         store.dispatch(
-                            ReviewQualityCheckAction.OpenLink(
-                                ReviewQualityCheckState.LinkType.ExternalLink(url),
-                            ),
+                            ReviewQualityCheckAction.OpenExplainerLearnMoreLink,
                         )
+                        onRequestDismiss()
```

Comment:
```
onRequestDismiss()
store.dispatch(ReviewQualityCheckAction.OpenExplainerLearnMoreLink)
```

We want to dismiss the sheet first and then navigate to a new tab, Same for action below.